ci/ci/build.py

```diff
@@ -1023,6 +1023,8 @@ def build(self, batch, code, scope):  # pylint: disable=unused-argument
                 parents=self.deps_parents(),
             )
 
+        n_cores = 1 if is_test_deployment else 4
```

Comment:
Do you mean to use 4 also for PRs in default CI? This will only use 1 if it is a deploy in a Dev CI

Reply:
I want 4 only for deploys in default. Is that what I am doing now?

Reply:
No, this is using 4 for deploys in default and PRs in default. I think you want

```
4 if params.scope == 'deploy' and not is_test_deployment else 1
```